adapters/rubicon/rubicontest/exemplary/simple-banner.json

```diff
@@ -299,9 +299,6 @@
             "id": "1",
             "bundle": "com.wls.testwlsapplication"
           },
-          "source": {
-            "pchain": "pchain"
```

Comment:
Nitpick: In order to make this test less verbose, can we remove `mockBidRequest.imp[0].bidder.pchain`?